recipe/meta.yaml

```diff
@@ -58,6 +58,6 @@ extra:
   recipe-maintainers:
     - mwcraig
     - bsipocz
-    - bernie-simon
+    - mcara
```

Comment:
@mcara , are you the maintainer now?

Reply:
Yes

Reply:
You will need to talk to Nadia about getting this package compatible with Python 3.12, if it is still used. FYI.

Reply:
> @mcara , are you the maintainer now?

My answer was for the `spherical_geometry` package.

Reply:
Right, you need to get the package itself compatible with Python 3.12 and then do a PyPI release. And then conda-forge would pick that up. Basically we cannot merge *this* PR.

Reply:
Actually it is `osx_64` but Python 3.12 for that arch is a subset of it, so still kinda apply.